WordPress/Classes/Models/Blog.m

```diff
@@ -64,8 +68,8 @@ - (void)didTurnIntoFault
     [super didTurnIntoFault];
 
     // Clean up instance variables
-    _blavatarUrl = nil;
-    _api = nil;
+    self.blavatarUrl = nil;
+    self.api = nil;
```

Comment:
`self.property = <value>;` does not hit the getter at all - it'd be a huge flaw in ObjC if that was the case.  It only hits the setter.

You can easily test this by setting a breakpoint there when running the code.